web/css/sidebar.css

```diff
@@ -42,6 +42,8 @@
 }
 
 #productsHolder footer .action {
+  font-family: 'Open Sans', sans-serif;
```

Comment:
What was the font stack for this element before? If this is what was being applied before I don't want to worry too much about it, but it seems a little short. Should it have "Arial", "Helvetica" or "Verdana" fallbacks for browsers that don't support web fonts or users who have web fonts disabled? Just a thought, but let's stick with whatever is being applied to this element in production for now.

Reply:
I agree, we should add 'Arial' in there. We are using this font stack consistently throughout so we'll keep it like this for now until we can change all references at the same time.